flask_monitoringdashboard/database/request.py

```diff
@@ -33,6 +35,10 @@ def get_date_of_first_request(db_session):
     return -1
 
 
+def create_time_based_sample_criterion(start_date: datetime, end_date: datetime):
```

Comment:
type hinting doesn't work in python 2

Reply:
Ah shit, I thought the requirement was py3. I'll fix that